app/src/main/java/org/mozilla/focus/activity/CustomTabActivity.kt

```diff
@@ -15,12 +18,15 @@ import org.mozilla.focus.ext.components
 class CustomTabActivity : MainActivity() {
     private lateinit var customTabId: String
 
-    private val customTabSession: Session by lazy {
-        components.sessionManager.findSessionById(customTabId)
+    private val customTabSession: SessionState by lazy {
+        components.store.state.findTabOrCustomTab(customTabId)
```

Comment:
Afaik this should be `findCustomTab()` and then we could use the `CustomTabSessionState` type? 🤔

Reply:
I tried that initially, but when we use 'Open in Focus' we crash because the tab is no longer a custom tab.

Reply:
We could have started to use the `migrate` use case, but this started opening up work in MainActivity and thought we might have conflicts if you had started working on that already.

Reply:
I'll put up a patch for `CustomTabsUseCases.remove()` it should only remove actual custom tabs. It currently can also remove normal tabs, but only because it still uses `SessionManager` under the hood.